### PR TITLE
[rsa2elk/sophos] Upgrade to ECS 8.0.0

### PR DESCRIPTION
**Pull request by adriansr**
Automated PR.

Upgrades rsa2elk package to ECS 8.0.0.

Includes manual changes to xg data stream from https://github.com/elastic/integrations/pull/2441

### CODE REVIEW

File: packages/sophos/data_stream/utm/fields/ecs.yml [hunk 1]
```diff
@@ -23,6 +23,7 @@
 - external: ecs
   name: destination.geo.country_name
 - description: Longitude and latitude.
+  level: core
```
Comment (P1llus) [inline packages/sophos/data_stream/utm/fields/ecs.yml:26]: Does this need to be here?

File: packages/sophos/data_stream/utm/fields/ecs.yml [hunk 2]
```diff
@@ -188,6 +187,7 @@
 - external: ecs
   name: source.geo.country_name
 - description: Longitude and latitude.
+  level: core
```
Comment (P1llus) [inline packages/sophos/data_stream/utm/fields/ecs.yml:190]: Same

File: packages/sophos/data_stream/xg/fields/ecs.yml [hunk 3]
```diff
@@ -25,6 +25,7 @@
 - external: ecs
   name: destination.geo.country_name
 - description: Longitude and latitude.
+  level: core
```
Comment (P1llus) [inline packages/sophos/data_stream/xg/fields/ecs.yml:28]: Same

File: packages/sophos/data_stream/xg/fields/ecs.yml [hunk 4]
```diff
@@ -192,6 +191,7 @@
 - external: ecs
   name: source.geo.country_name
 - description: Longitude and latitude.
+  level: core
```
Comment (P1llus) [inline packages/sophos/data_stream/xg/fields/ecs.yml:194]: Same

**Review by P1llus — APPROVED** [review]
LGTM, just a small comment

File: packages/sophos/data_stream/utm/fields/ecs.yml (hunk 1 above)
Comment (adriansr) [reply packages/sophos/data_stream/utm/fields/ecs.yml]: Ups, you're right. That was introduced by some automation

**Pull request merged**